platform/ios/docs/guides/Working with Style Functions.md

```diff
@@ -4,15 +4,15 @@
   Edit platform/darwin/scripts/generate-style-code.js, then run `make darwin-style-code`.
 -->
 
-# Data-Driven Styling
+# Working with Style Functions
```

Comment:
This guide talks about using style functions in conjunction with the runtime styling API, as opposed to using style functions in style JSON. Perhaps we should call this guide “Using Style Functions at Runtime”.